src/test/java/team/asd/entities/TestLocation.java

```diff
@@ -0,0 +1,20 @@
+package team.asd.entities;
+
+public class TestLocation implements IsLocation {
+	private final String name;
+	private final String city;
+	private final String street;
+	private final String house;
+
+	public TestLocation(String city, String street, String house) {
+		this.city = city;
+		this.street = street;
+		this.house = house;
+		this.name = city + " " + street + ", " + house;
```

Comment:
`this.name = city + " " + street + ", " + house;`
you can move this logic to toString() method, 
don't need to use field  'name' for that